disnake/interactions/base.py

```diff
@@ -163,7 +163,12 @@ def _from_data(self, data: InteractionPayload):
             except KeyError:
                 pass
             else:
-                self.author = Member(state=self._state, guild=guild, data=member)  # type: ignore
+                author_id = int(member["user"]["id"])  # type: ignore
+                self.author = (
+                    guild
+                    and guild.get_member(author_id)  # type: ignore
```

Comment:
why not `guild.get_member(int(member["user"]["id"]))`?

Reply:
I didn't want to make it too long, I bet black will complain about it.

Reply:
black will format if it its too long for you :D

Reply:
We already decided to not do that.